renovate.json

```diff
@@ -18,11 +18,14 @@
         "*"
       ],
       "rangeStrategy": "pin"
+    },
+    {
+      "depTypeList": ["engines"],
+      "rangeStrategy": "auto"
     }
   ],
   "stabilityDays": 7,
   "node": {
-    "enabled": true,
```

Comment:
I guess this is removed since it's essentially the default?

Reply:
Correct, it's enabled by default. I think